cpp/src/parquet/column_reader.cc

```diff
@@ -101,6 +101,12 @@ inline void CheckNumberDecoded(int64_t number_decoded, int64_t expected) {
                                    std::to_string(expected));
   }
 }
+
+constexpr std::string_view kErrorRepDefLevelNotMatchesNumValues =
+    "Number of decoded rep / def levels did more or less than num_values in page_header";
+constexpr std::string_view kErrorRepDefLevelInEqual =
+    "Number of decoded rep / def levels did not match";
```

Comment:
Is this message still useful? It seems all cases should be covered by `kErrorRepDefLevelNotMatchesNumValues` above.